docs/plugins/storage.md

```diff
@@ -0,0 +1,62 @@
+# Storage
+
+You can back your stores with localStorage or sessionStorage by including the `NgxsStoragePlugin` plugin.
+
+NOTE: all options are OPTIONAL and are shown with their defaults
+
+```TS
+import { NgxsModule } from '@ngxs/store';
+import { NgxsLocalStoragePluginModule, StorageOption } from '@ngxs/storage-plugin';
+
+@NgModule({
+  imports: [
+    NgxsModule.forRoot([]),
+    NgxsLocalStoragePluginModule.forRoot({
+      /**
+       * Default key to persist. You can pass a string or array of string
+       * that can be deeply nested via dot notation.
+       */
+      key: '@@STATE',
+
+      /**
+       * Storage strategy to use. Thie defaults to LocalStorage but you
+       * can pass SessionStorage or anything that implements the localStorage API.
+       */
+      storage: StorageOption.LocalStorage,
+
+      /**
+       * Custom deseralizer. Defaults to JSON.parse
+       */
+      deserialize: JSON.parse,
```

Comment:
@un33k - If passing a different storage engine breaks, wouldn't this break too?

Reply:
this is actually something I am looking into now. writing tests around a simple custom storage option now

Reply:
@amcdnl so it will break depending on what you do with your implementation. it is easy enough to pass a custom serializer/deserialzer though so im not to concerned. maybe a documentation issue more then anything else